enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -0,0 +1,219 @@
+---
+title: hypershift-audit-logs
+authors:
+  - aconway@redhat.com
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2023-03-31
+tracking-link:
+  - TBD
+see-also:
+  - /enhancements/cluster-logging/api-audit-log-policy.md
+---
+
+# Hypershift Audit Logs
+
+## Summary
+
+In a self contained cluster, users can forward application, infrastructure and audit logs.
+In a hypershift cluster, API server audit logs do not exist on the guest cluster,
+they are generated in Hypershift Control Plane (HCP) pods on the management cluster.
+
+This proposal deploys the existing cluster logging operator (CLO) on the management cluster.
+It will forward HCP audit logs according to a policy and destination configured on the guest cluster,
+and independently forward audit logs with the policy/ destination required by hypershift SD.
+
+**NOTE**: This proposal depends on [api-audit-log-policy](/api-audit-log-policy.md)
+
+## Motivation
+
+- Allow users and SREs with conflicting requirements to get the logs they need.
+- Forward API server logs from the management cluster, as they are not available on the guest cluster.
+- Replace the SD Splunk forwarder with the CLO which is more flexible and feature-rich.
+- Centralize log forwarding in the logging project, give hypershift team control of their logging needs.
+
+### User Stories
+
+#### As a cluster owner, I want to store and/or forward selected API audit logs to my chosen destinations.
+
+- configure API audit policy and forwarding using guest cluster resource.
+- have the desired audit logs forwarded from the management cluster to my destination.
+
+#### As a Hypershift SRE, I want to control plane audit logs to go to my chosen destination with my policy.
+
+- configure the SD internal audit policy
+- forward logs to the SD internal destination (splunk)
+- independent of user configuration.
+
+### Goals
+
+- Multiple separate audit policies. 
+  See [api-audit-log-policy](/api-audit-log-policy.md) for how the CLO will support this.
+- Control log forwarding from the _management_ cluster with configuration in the _guest_ cluster.
+- Keep control of all management cluster forwarding on the management cluster.
+  - guest configuration requests forwarding, management cluster controls what is delivered.
+
+### Non-Goals
+
+- Only API-server audit logs, no application, infrastructure or node audit logs.
+- May not support every possible log destination or configuration.
+  There may be restrictions imposed by the management cluster.
```

Comment:
Is this restriction enforced by what is exposed in `HypershiftLogForwarder` or is there some other configuration option available to the service provider to further restrict functionality?